packages/applications-service-api/.dockerignore

```diff
@@ -0,0 +1 @@
+**/*.spec.js
```

Comment:
might want to remove this.... prevents any "spec.js" files from being deployed  as part of the docker image, which makes it difficult to run them against the images as part of the deployment pipeline.